File: majorityelements.cpp
#include<iostream>
#include<vector>
#include<map>
using namespace std;

int main ()
{

std::vector<int> nums;
nums.push_back(3);
nums.push_back(5);
nums.push_back(3);
map <int,int> mp;

for (int i = 0; i< nums.size() ;i++)
{
  mp[nums[i]]++;
}

for(int i=0;i<nums.size();i++)
{
  if (mp[nums[i]] > (nums.size()/2))
  {
    cout << nums[i];
  }
}








}

File: majorityelements.py
nums = [4,5,4]
a ={}
for i in range(len(nums)):
    if nums[i] not in a:
        a[nums[i]] =1
    else:
        a[nums[i]] += 1

for i in range (len(nums)):
    if(a[nums[i]] > (len(nums)/2)):
        print(nums[i])
        break
